src/component/pages/objectIdentifierPage/objectIdentifierPage.tsx

```diff
@@ -0,0 +1,123 @@
+        videoRef.current.srcObject = mediaStream;
+      }
+    } catch (error) {
+      console.error('Error accessing camera:', error);
+    }
+  };
+
+  const startRecording = () => {
+    if (!stream) return;
+    
+    const mediaRecorder = new MediaRecorder(stream);
+    mediaRecorderRef.current = mediaRecorder;
+    
+    mediaRecorder.ondataavailable = (event) => {
+      if (event.data.size > 0) {
+        setRecordedChunks((prev) => [...prev, event.data]);
+      }
+    };
+    
+    setStartTime(new Date()); 
+    setEndTime(null);
+    setDuration(null);
+    setPrediction(null);
+    setLoading(false); // Reset loading state
+
+    mediaRecorder.start();
+  };
+
+  const stopRecording = () => {
+    if (mediaRecorderRef.current) {
+      mediaRecorderRef.current.stop();
+      const end = new Date();
+      setEndTime(end);
+
+      if (startTime) {
+        const recordedDuration = (end.getTime() - startTime.getTime()) / 1000;
+        setDuration(recordedDuration);
+      }
+    }
+  };
+
+  const submitVideo = async () => {
+    if (recordedChunks.length === 0) return;
+
+    const blob = new Blob(recordedChunks, { type: 'video/webm' });
+    const formData = new FormData();
+    formData.append('file', blob, 'recording.webm');
+
+    setLoading(true); // Show loading animation
+
+    try {
+      const response = await fetch('http://127.0.0.1:2220/predict/', {
+        method: 'POST',
+        body: formData,
+      });
+
+      if (response.ok) {
+        const result = await response.json();
+        setPrediction(result.predicted_sign); // Set prediction result
+      } else {
+        alert('Failed to upload video');
+      }
+    } catch (error) {
+      console.error('Error uploading video:', error);
+    } finally {
+      setLoading(false); // Hide loading animation when response is received
+    }
+  };
+
+  return (
+    <div>
+      <h1>Welcome to the Object Identifier Page</h1>
+      <video ref={videoRef} autoPlay playsInline style={{ width: '100%', maxWidth: '600px' }}></video>
+      
+      <div>
+        <button onClick={openCamera}>Open Camera</button>
+        <button onClick={startRecording}>Start Recording</button>
+        <button onClick={stopRecording}>Stop Recording</button>
+        <button onClick={submitVideo}>Submit</button>
+      </div>
+
+      <div style={{ marginTop: '20px' }}>
+        {startTime && <p>📍 Start Time: {startTime.toLocaleTimeString()}</p>}
+        {endTime && <p>🛑 End Time: {endTime.toLocaleTimeString()}</p>}
+        {duration !== null && <p>⏳ Duration: {duration.toFixed(2)} seconds</p>}
+      </div>
+
+      {/* Show Loading Animation */}
+      {loading && (
+        <div style={{ marginTop: '20px', fontSize: '18px', fontWeight: 'bold', color: 'blue' }}>
+          ⏳ Processing... Please wait...
+        </div>
+      )}
+
+      {/* Show Prediction Result */}
+      {prediction && !loading && (
+        <p style={{ fontSize: '18px', fontWeight: 'bold', color: 'green' }}>
```

Comment:
```suggestion
        <p style={{ fontSize: '18px', fontWeight: 'bold', color: 'green' }}>
```
i think if you can create ts css file and move to all css it. it's better way